.github/workflows/ci.yml

```diff
@@ -17,6 +17,9 @@ jobs:
         run: shards install
       - name: Format
         run: crystal tool format --check
+      - name: Crystal Ameba Linter
+        run: crystal bin/ameba.cr
```

Comment:
oh, I wonder if this should just be https://github.com/luckyframework/lucky/blob/master/.github/workflows/ci.yml#L21 ?

Reply:
I think either one will work, but I'm all for keeping things consistent among the org's repos. Changed!